api/db-cdr/build.gradle

```diff
@@ -1,21 +1,25 @@
-buildscript {
-    repositories {
-        mavenCentral()
-    }
-    dependencies {
-        classpath 'org.liquibase:liquibase-gradle-plugin:1.2.4'
-        classpath 'mysql:mysql-connector-java:5.1.37'
-    }
+plugins {
+    id 'application'
+    id 'org.liquibase.gradle' version '2.1.0'
+}
+
+repositories {
+    mavenCentral()
 }
-apply plugin: 'org.liquibase.gradle'
-apply plugin: 'application'
 
 def db_host = System.getenv("DB_HOST") ?: "db"
 def db_port = System.getenv("DB_PORT") ?: "3306"
 def db_name = System.getenv("CDR_DB_NAME") ?: "cdr"
 def liquibase_password = System.getenv("LIQUIBASE_DB_PASSWORD") ?: "lb-notasecret"
 applicationDefaultJvmArgs = ["-Xmx2048m","-Xms2048m"]
 
+dependencies {
+    liquibaseRuntime 'org.liquibase:liquibase-core:4.5.0'
+    liquibaseRuntime 'org.liquibase:liquibase-groovy-dsl:3.0.0'
+    liquibaseRuntime 'info.picocli:picocli:4.6.1'
```

Comment:
Where did this dep come from? Is this required by the gradle plugin?

Reply:
I got it from https://github.com/liquibase/liquibase-gradle-plugin#2-setting-up-the-classpath
still trying to make it working...